.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.231.6/containers/go
+{
+	"name": "Go",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update the VARIANT arg to pick a version of Go: 1, 1.18, 1.17
+			// Append -bullseye or -buster to pin to an OS version.
+			// Use -bullseye variants on local arm64/Apple Silicon.
+			"VARIANT": "1.17",
+			// Options
+			"NODE_VERSION": "none"
```

Comment:
Also this is redundent?

Reply:
Yes, this is redundant. I remove this. And we remove Nodejs in Dockerfile because we are not using it.